cachito/web/models.py

```diff
@@ -180,13 +189,14 @@ def to_json(self):
     def from_json(cls, kwargs):
         # Validate all required parameters are present
         required_params = {'repo', 'ref', 'pkg_managers'}
-        missing_params = required_params - set(kwargs.keys())
+        optional_params = {'flags'}
+        missing_params = required_params - set(kwargs.keys()) - optional_params
         if missing_params:
             raise ValidationError('Missing required parameter(s): {}'
                                   .format(', '.join(missing_params)))
 
         # Don't allow the user to set arbitrary columns or relationships
-        invalid_params = kwargs.keys() - required_params
+        invalid_params = kwargs.keys() - required_params - optional_params
```

Comment:
To compute `missing_params`, `kwargs.keys()` is converted into a `set`. Why not do it here also?